README.md

```diff
@@ -5,117 +5,91 @@
 [![CircleCI](https://circleci.com/gh/stoplightio/spectral.svg?style=svg)](https://circleci.com/gh/stoplightio/spectral)
 
-A flexible JSON object linter with out of the box support for OpenAPI v2 and v3
+A flexible JSON linter with out of the box support for OpenAPI v2 and v3.
+
+![Demo of Spectral linting an OpenAPI document from the CLI](./docs/demo.svg)
 
 ## Features
 
-- Create custom rules to lint _any JSON object_
-- Use JSON paths to apply rules / functions to specific parts of your JSON objects
-- Built-in set of functions to help [build custom rules](#creating-a-custom-rule). Functions include pattern checks, parameter checks, alphabetical ordering, a specified number of characters, provided keys are present in an object, etc
-- [Create custom functions](#creating-a-custom-function) for advanced use cases
-- Optional ready to use rules and functions to validate and lint [OpenAPI v2 _and_ v3 documents](#example-linting-an-openapi-document)
-- Validate JSON with [Ajv](https://github.com/epoberezkin/ajv)
+- Create custom rules to lint JSON or YAML objects
+- Ready to use rules to validate and lint OpenAPI v2 _and_ v3 documents
+- Use JSON path to apply rules to specific parts of your objects
+- Built-in set of functions to help [create custom rules](./docs/rulesets.md). Functions include pattern checks, parameter checks, alphabetical ordering, a specified number of characters, provided keys are present in an object, etc.
+- Create custom functions for advanced use cases
+- Validate JSON with [Ajv](https://www.npmjs.com/package/ajv)
 
 ## Installation
 
-### Local Installation
-
-```bash
-npm install @stoplight/spectral
-```
-
-### Global Installation
+Spectral's CLI can be installed via npm or yarn.
 
 ```bash
 npm install -g @stoplight/spectral
-```
-
-Supports Node v8.3+ and modern browsers.
-
-### Executable binaries
-
-For users without Node and/or NPM/Yarn, we provide standalone packages for all major platforms. We also provide a shell script to auto download the executable based on your operating system:
-
-`curl -L https://raw.githack.com/stoplightio/spectral/master/install.sh | sh`
-
-Note, the binaries are *not* auto-updatable, therefore you will need to download a new version on your own.
-
-#### Installing binaries system-wide
 
-##### Linux
+# OR
 
-```bash
-sudo mv ./spectral-linux /usr/local/bin/spectral
+yarn global add @stoplight/spectral
 ```
 
-You may need to restart your terminal.
-Now, `spectral` command will be accessible in your terminal.
-
-Head over to [releases](https://github.com/stoplightio/spectral/releases) for the latest binaries.
+Spectral is avaiable as a Docker image as well.
 
-
-### Docker
 ```bash
-docker run --rm -it stoplight/spectral lint "${URL}"`
+docker run --rm -it stoplight/spectral lint "${url or filepath to openapi file}"`
```

Comment:
This is not correct, in case you want to use a file on your local computer you effectively need to mount a volume in the container. I'll change the command.